### PR TITLE
Remove jQuery as a dependency for setting up channels

### PR DESCRIPTION
**Pull request by skippednote**
The channels setup is already using ES2015 and the DOM API has become simple enough to understand now.

### CODE REVIEW

**Comment by Gazler** [comment]
I am :+1: on removing JQuery. Will double check the code and get this merged.

**Comment by Gazler** [comment]
Thanks <3

File: G_channels.md [hunk 1]
```diff
@@ -177,18 +165,20 @@ All we had to do is detect that enter was pressed and then `push` an event over
...
-let messagesContainer = $("#messages")
+let chatInput         = document.querySelector("#chat-input")
+let messagesContainer = document.querySelector("#messages")
 
-chatInput.on("keypress", event => {
+chatInput.addEventListener("keypress", event => {
   if(event.keyCode === 13){
-    channel.push("new_msg", {body: chatInput.val()})
+    channel.push("new_msg", {body: chatInput.value})
     chatInput.val("")
```
Comment (mydearxym) [inline G_channels.md]: should't be chatInput.value = ""  ?

Comment (Gazler) [inline G_channels.md]: Great spot

**Pull request merged**